src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/domain/handlers/common/EventInformation.java

```diff
@@ -35,15 +38,15 @@
     private final String userId;
 
     @JsonCreator
-    public EventInformation(@JsonProperty("eventInstanceId") String eventInstanceId,
-                            @JsonProperty("eventTimeStamp")LocalDateTime eventTimeStamp,
-                            @JsonProperty("caseId") String caseId,
-                            @JsonProperty("jurisdictionId") String jurisdictionId,
-                            @JsonProperty("caseTypeId") String caseTypeId,
-                            @JsonProperty("eventId") String eventId,
-                            @JsonProperty("previousStateId") String previousStateId,
-                            @JsonProperty("newStateId") String newStateId,
-                            @JsonProperty("userId") String userId) {
+    public EventInformation(@JsonProperty("EventInstanceId") String eventInstanceId,
+                            @JsonProperty("EventTimeStamp")LocalDateTime eventTimeStamp,
+                            @JsonProperty("CaseId") String caseId,
+                            @JsonProperty("JurisdictionId") String jurisdictionId,
+                            @JsonProperty("CaseTypeId") String caseTypeId,
+                            @JsonProperty("EventId") String eventId,
+                            @JsonProperty("PreviousStateId") String previousStateId,
+                            @JsonProperty("NewStateId") String newStateId,
+                            @JsonProperty("UserId") String userId) {
```

Comment:
Should User_Id be removed?

Reply:
It should be there as part of the request. But, when logging onto dead letter Q, it should be removed which I did.